zerver/webhooks/freshstatus/doc.md

```diff
@@ -0,0 +1,79 @@
+Receive Freshstatus notifications in Zulip!
+
+1. {!create-stream.md!}
+
+1. {!create-bot-construct-url-indented.md!}
+
+1. Go to your Freshstatus dashboard. Click **Settings** and then
+   **Integrations**. Go to **Webhooks** and click **Manage** and
+   then click **New Webhook**.
```

Comment:
This should be:

On your Freshstatus dashboard, click **Settings** and click**Integrations**. Go to **Webhooks** and click **Manage**. Click **New Webhook** to set up a new webhook.